database/postgres/pg.js

```diff
@@ -61,4 +61,22 @@ pgModule.deletePgAccount = async (username) => {
   }
 };
 
+pgModule.userHasPgAccount = async (username) => {
+  if (!username) return false;
+  const usernameAlphabetical = username.replace("@", "").replace(".", "");
```

Comment:
https://github.com/garageScript/databases/pull/179/files

In this PR, I changed how we encode username. So we don't use `usernameAlphabetical` variable anymore. You can simply put `username`